elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/Column.java

```diff
@@ -41,11 +43,16 @@
      * @throws NullPointerException if {@code field} or {@code fieldType} is {@code null}
      */
     public Column(Schema schema, String field, Meta annotation, Class<?> fieldType) {
+        this(schema, field, annotation, fieldType, field);
+    }
+
+    public Column(Schema schema, String field, Meta annotation, Class<?> fieldType, String columnName) {
         this.name = Objects.requireNonNull(field, "field");
         this.longName = annotation == null || annotation.longName().isEmpty() ? field : annotation.longName();
         this.description = annotation == null || annotation.description().isEmpty() ? field : annotation.description();
         this.dataType = Objects.requireNonNull(fieldType, "fieldType");
         this.schema = schema;
+        this.columnName = Objects.requireNonNull(columnName, "field");
```

Comment:
This looks wrong, should it be `this.columnName = columnName == null ? this.name : columnName`?